### ISSUE TITLE
Sparse UAV yaw actions cause extreme quantile-normalized values and π0.5 training loss spikes

### ISSUE THREAD

**Issue by amzking2** [SpencerSon2001/AIR-VLA#6]
Hello, and thank you for releasing the AIR-VLA dataset and code.

While full-parameter fine-tuning OpenPI π0.5 on AIR-VLA, I found that drone_dw, index 3 of the 12D action vector, produces extreme values under quantile normalization and causes intermittent explosions in both training loss and gradient norm.

In the released scripts/convert_all_to_hdf5.py, yaw increments are first converted from radians to degrees:
```python
drone_dw = np.rad2deg(df["drone_dw"].values)
```
Three consecutive 60 Hz frames are then accumulated into one 20 Hz action. Therefore, a valid yaw command of 0.005 rad per frame becomes:
```
0.005 × 3 × 180 / π ≈ 0.859°
```
My converted training dataset contains 577 episodes and 287,799 samples. With the default π0.5 quantile normalization:
```python
normalized = (x - q01) / (q99 - q01 + 1e-6) * 2 - 1
```
the statistics for action dimension 3 are:
```
raw_min = -0.85944575
raw_max = +0.85943723
q01 = -0.00034803833
q99 = -0.0000042617321
q99 - q01 = 0.0003437766
```
Most frames contain no yaw command, while valid turning actions account for only about 0.55% of the data. As a result, both positive and negative yaw commands fall outside the 1st–99th percentile interval and are normalized to approximately ±4986. A full scan found 1,588 action elements satisfying:
```
|normalized yaw| > 1000
```
This causes the following behavior during training:
```
Normal steps:
loss ≈ 0.43–0.52
grad_norm ≈ 4–6

Spike steps:
loss=28611.54, grad_norm=1086.23
```
The loss immediately returns to its normal range afterward, suggesting that the spikes are caused by batches containing extreme normalized yaw targets rather than global model divergence.

Did the π0/π0.5 experiments in the paper use quantile normalization? Were normalization statistics computed separately for each task rather than over the merged multi-task dataset? Was any manual scaling, clipping, or special normalization applied to the sparse drone_dw dimension? Could you release the training configuration and normalization statistics used in the paper? 

The public repository appears to contain data collection, HDF5 conversion, and evaluation code, but not the training entry point or normalization configuration, so clarification on the original preprocessing pipeline would be very helpful. Thank you.
